elasticsearch5/docker/confd/templates/elasticsearch.config.tmpl

```diff
@@ -30,3 +30,14 @@ node.data: false
 {{if exists "/elasticsearch/gateway/expected/nodes" }}gateway.expected_nodes: {{ getv "/elasticsearch/gateway/expected/nodes" }}{{end}}
 {{if exists "/elasticsearch/gateway/recover/after/time" }}gateway.recover_after_time: {{ getv "/elasticsearch/gateway/recover/after/time" }}{{end}}
 {{if exists "/elasticsearch/gateway/recover/after/nodes" }}gateway.recover_after_nodes: {{ getv "/elasticsearch/gateway/recover/after/nodes" }}{{end}}
+
+
+xpack.ssl.key:           /usr/share/elasticsearch/config/key.pem
+xpack.ssl.certificate:   /usr/share/elasticsearch/config/cert.pem
+{{ if eq "TRUE" (toUpper (getv "/elasticsearch/usessl")) }}
+xpack.security.transport.ssl.enabled: true
+xpack.security.http.ssl.enabled: true
+{{ else }}
+xpack.security.transport.ssl.enabled: false
+xpack.security.http.ssl.enabled: false
+{{end}}
```

Comment:
We get default authorisation too. use
`curl -k -u elastic:changeme https://localhost:9200`